lib/window.ts

```diff
@@ -8,25 +8,66 @@ export default function createWindow(currentWindow: Window, channel: Channel): W
   const autoUpdateHeight = () => {
     self.updateHeight()
   }
-  const observer = new MutationObserver(autoUpdateHeight)
+
+  const heightObserverCallback = (mutations: Array<MutationRecord>) => {
+    checkAbsolutePositionedElems(mutations)
+    if (isAutoResizing) {
+      autoUpdateHeight()
+    }
+  }
+
+  const observer = new MutationObserver(heightObserverCallback)
   let oldHeight: number
   let isAutoResizing = false
+  let checkAbsoluteElements = false
+  const absolutePositionedElems: Set<Element> = new Set()
+
+  // Start observer to get absolute elements
+  observer.observe(document.body, {
+    attributes: true,
+    childList: true,
+    subtree: true,
+    characterData: true,
+  })
```

Comment:
Should we clean up after this listener on stop?

Reply:
We never stop the observer. The observer starts when initializing the sdk.

Reply:
Yeah, that was broadly m question actually. Don't we want to start/stop listening only if needed? To not introduce performance regressions in pages with tens of Apps

Reply:
We do start immediately in order to not miss any absolute element. If we only start listening when calling `startAutoResizer`, we will have to go through the whole dom. Also, if the user wants to use `updateHeight()` instead of the auto resizer, we would also need to go through the whole dom every time.